### PR TITLE
HPCC4J-631 Provide meaningful version error message

### PR DESCRIPTION
**Pull request by rpastrana**
- Ensures version response is not HTML based
- Adds relevant message to parsing errors

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] This change does not cause any existing JUnits to fail.
  - [ ] I have include JUnit coverage to test this change
  - [ ] I have performed system test and covered possible regressions and side effects.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] This change fixes the problem, not just the symptom

### CODE REVIEW

**Comment by rpastrana** [comment]
we might need to add a test without credentials which would cause the ESP to respond w/ HTML, but it would require a known secured HPCC test environment

File: wsclient/src/main/java/org/hpccsystems/ws/client/BaseHPCCWsClient.java [hunk 1]
```diff
@@ -159,24 +160,43 @@ private String getTargetHPCCBuildVersionString() throws Exception
         if (wsconn == null)
             throw new Exception("Cannot get target HPCC build version, client connection has not been initialized.");
 
-        String response = wsconn.sendGetRequest("WsSMC/Activity?rawxml_");//throws
+        String response = wsconn.sendGetRequest("WsSMC/Activity?rawxml_");//throws IOException if http != ok
 
         if (response == null || response.isEmpty())
             throw new Exception("Cannot get target HPCC build version, received empty " + wsconn.getBaseUrl() + " wssmc/activity response");
 
+        if (response.startsWith("<html")) //crude, but can prevent wasteful overhead
```
Comment (jpmcmu) [inline wsclient/src/main/java/org/hpccsystems/ws/client/BaseHPCCWsClient.java:168]: Should this handle possible whitespace before the html tag?

**Review by jpmcmu — APPROVED** [review]
@rpastrana Looks pretty good, minor question about handling whitespace, but not critical IMO

**Comment by rpastrana** [comment]
@jpmcmu made the change we discussed, please give it a quick look-over

**Review by jpmcmu — APPROVED** [review]
@rpastrana looks good

**Pull request merged**